src/mjolnir/valhalla_fetch_transit.cc

```diff
@@ -28,8 +28,8 @@
 
 #include "filesystem.h"
```

Comment:
it seems like this file should be unchanged right? can you revert it and leave it untouched. once we have raw gtfs working properly we can completely delete it but it should probably live for a while yet, untouched

Reply:
I will have to change removing `transitpbf.h` and including `ingest_transit.h` to not get build errors because the first file was removed and the functions were moved to the second file, but I will revert back other changes.

Reply:
Ah OK yeah changing includes is fine just revert the other diffs

Reply:
Yup, reverted all other edits